integration_test.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+set -Eemo pipefail
+# This script is meant to be run inside https://hub.docker.com/_/docker
+# See .github/workflows/ci.yml:18 for how to run this yourself
```

Comment:
Instead of referring to a specific line number (which may change over time), how about using [Tagref](https://github.com/stepchowfun/tagref)? You can add a tag (e.g., `[tag:integration_test_step]` or something similar) to the relevant line in `.github/workflows/ci.yml` and then reference it from here. You don't have to install Tagref or anything, since the Toastfile already uses it (in the `lint` target).

Let me know if you have any questions. I know Tagref will probably be new to you since it's not a very well-known tool (sadly!).